content/posts/2019-08-03---Going-from-ember-to-vue.md

```diff
@@ -162,10 +175,16 @@ This means that child component does not need to receive an action as a paramete
 All it does is emit something, pass any properties along with it and then it stops caring.
 It's up to the parent component to listen to it and respond.
 
-One of the things I disliked with the actions helper in Ember was when the child component could call `this.passedAction()` and I would get an error that it wasn't a function because I forgot to pass it down, the child component stopped working.
-With emitting, the child component can continue to function.
-To be clear: making API calls to your backend should happen through the Vuex store -- mutating applicaton state should not be done with `emit`.
-However, if you have buttons component that toggles something on and off, this is a good case for using `emit` as the parent component is most likely the only thing that cares about the event.
+One of the things I disliked with the actions helper in Ember was that a child component might not always want to bubble something up.
+Say you had a component like this in Ember: `<AuthorInfo @updateProfile={{action "updateProfile"}} />`.
```

Comment:
```hbs
<AuthorInfo @updateProfile={{this.updateProvide}} />
```

Reply:
Octane is not out yet. Classic components expect the syntax in the post.

Reply:
But you can use `fn` helper `<AuthorInfo @updateProfile={{fn this.updateProvide}} />`, can't you?

Reply:
You can, but it's doing nothing in your code sample, since you are not passing extra parameters.